src/ngraph_backend_manager.cc

```diff
@@ -40,6 +40,8 @@ std::map<std::string, int> BackendManager::ref_count_each_backend_;
 
 Status BackendManager::SetBackendName(const string& backend_name) {
   std::lock_guard<std::mutex> lock(BackendManager::ng_backend_name_mutex_);
+  // TODO: Should NGRAPH_TF_BACKEND be read here as well and should it override
+  // backend_name?
```

Comment:
I would think No, if we look at environment variables as means to override some settings. GetCurrentlySetBackendName() returns the backend that is requested by the environment variable.